### ISSUE TITLE
FileNotFoundError During Finetuning in "Large-scale Scene Reconstruction with Partitioning and LoD"

### ISSUE THREAD

**Issue by tan-tao11** [yzslab/gaussian-splatting-lightning#87]
Hi,
Thanks for your excellent work on this project. However, I encountered a FileNotFoundError during the finetuning stage. Below are the details of the issue:

Command:
python utils/finetune_pruned_partitions_v2.py \
    data/zhangzhou_colmap_cuda/partitions-size_3.0-enlarge_0.1-visibility_0.9_0.1 \
    -p zhangzhou_largegs-pruned \
    -t zhangzhou_largegs \
    --config configs/appearance_embedding_renderer/sh_view_dependent.yaml

Logs:
['-02_-02', '-01_-02', '000_-02', '001_-02', '-02_-01', '-01_-01', '000_-01', '001_-01', '-02_000', '-01_000', '000_000', '001_000', '-02_001', '-01_001', '000_001', '001_001']
  0%|                                                                                                                                                                            | 0/16 [00:00<?, ?it/s]max_steps:  29000
partition_image_number:  176
extra_epoches:  30
scale_mode:  linear
['python', 'main.py', 'fit', '--config=configs/appearance_embedding_renderer/sh_view_dependent.yaml', '--config=/data1/tantao/weizhi/code/gaussian-splatting-lightning/outputs/zhangzhou_largegs/-02_-02/config.yaml', '--max_steps', '29000', '--model.gaussian.optimization.means_lr_scheduler.init_args.max_steps', '39000', '--model.density.densification_interval', '100', '--model.density.opacity_reset_interval', '3000', '--model.density.densify_from_iter', '500', '--model.density.densify_until_iter', '15000', '-n=-02_-02', '--data.path', '/data/tantao/weizhi/data/zhangzhou_colmap_cuda', '--project', 'zhangzhou_largegs-pruned', '--output', '/data1/tantao/weizhi/code/gaussian-splatting-lightning/outputs/zhangzhou_largegs-pruned', '--logger', 'tensorboard', '--ckpt_path=/data1/tantao/weizhi/code/gaussian-splatting-lightning/outputs/zhangzhou_largegs/-02_-02/pruned_checkpoints/latest-opacity_pruned-0.6.ckpt']
/data1/tantao/miniconda3/envs/gspl/lib/python3.9/site-packages/pydantic/_migration.py:283: UserWarning: `pydantic.utils:Representation` has been removed. We are importing from `pydantic.v1.utils:Representation` instead.See the migration guide for more details: https://docs.pydantic.dev/latest/migration/
  warnings.warn(
usage: main.py [options] fit [-h] [-c CONFIG] [--print_config[=flags]] [--seed_everything SEED_EVERYTHING] [--trainer CONFIG] [--trainer.accelerator.help CLASS_PATH_OR_NAME]
                             [--trainer.accelerator ACCELERATOR] [--trainer.strategy.help CLASS_PATH_OR_NAME] [--trainer.strategy STRATEGY] [--trainer.devices DEVICES]
                             [--trainer.num_nodes NUM_NODES] [--trainer.precision PRECISION] [--trainer.logger.help CLASS_PATH_OR_NAME] [--trainer.logger LOGGER]
                             [--trainer.callbacks.help CLASS_PATH_OR_NAME] [--trainer.callbacks CALLBACKS] [--trainer.fast_dev_run FAST_DEV_RUN] [--trainer.max_epochs MAX_EPOCHS]
                             [--trainer.min_epochs MIN_EPOCHS] [--trainer.max_steps MAX_STEPS] [--trainer.min_steps MIN_STEPS] [--trainer.max_time MAX_TIME]
                             [--trainer.limit_train_batches LIMIT_TRAIN_BATCHES] [--trainer.limit_val_batches LIMIT_VAL_BATCHES] [--trainer.limit_test_batches LIMIT_TEST_BATCHES]
                             [--trainer.limit_predict_batches LIMIT_PREDICT_BATCHES] [--trainer.overfit_batches OVERFIT_BATCHES] [--trainer.val_check_interval VAL_CHECK_INTERVAL]
                             [--trainer.check_val_every_n_epoch CHECK_VAL_EVERY_N_EPOCH] [--trainer.num_sanity_val_steps NUM_SANITY_VAL_STEPS] [--trainer.log_every_n_steps LOG_EVERY_N_STEPS]
                             [--trainer.enable_checkpointing {true,false,null}] [--trainer.enable_progress_bar {true,false,null}] [--trainer.enable_model_summary {true,false,null}]
                             [--trainer.accumulate_grad_batches ACCUMULATE_GRAD_BATCHES] [--trainer.gradient_clip_val GRADIENT_CLIP_VAL] [--trainer.gradient_clip_algorithm GRADIENT_CLIP_ALGORITHM]
                             [--trainer.deterministic DETERMINISTIC] [--trainer.benchmark {true,false,null}] [--trainer.inference_mode {true,false}] [--trainer.use_distributed_sampler {true,false}]
                             [--trainer.profiler.help CLASS_PATH_OR_NAME] [--trainer.profiler PROFILER] [--trainer.detect_anomaly {true,false}] [--trainer.barebones {true,false}]
                             [--trainer.plugins.help CLASS_PATH_OR_NAME] [--trainer.plugins PLUGINS] [--trainer.sync_batchnorm {true,false}]
                             [--trainer.reload_dataloaders_every_n_epochs RELOAD_DATALOADERS_EVERY_N_EPOCHS] [--trainer.default_root_dir DEFAULT_ROOT_DIR] [--model CONFIG]
                             [--model.light_gaussian CONFIG] [--model.light_gaussian.prune_steps PRUNE_STEPS] [--model.light_gaussian.prune_decay PRUNE_DECAY]
                             [--model.light_gaussian.prune_percent PRUNE_PERCENT] [--model.light_gaussian.prune_type v_important_score] [--model.light_gaussian.v_pow V_POW]
                             [--model.gaussian.help CLASS_PATH_OR_NAME] [--model.gaussian GAUSSIAN] [--model.background_color [ITEM,...]] [--model.random_background {true,false}]
                             [--model.output_path OUTPUT_PATH] [--model.save_val_output {true,false}] [--model.save_val_metrics {true,false,null}] [--model.max_save_val_output MAX_SAVE_VAL_OUTPUT]
                             [--model.renderer.help CLASS_PATH_OR_NAME] [--model.renderer RENDERER] [--model.metric.help CLASS_PATH_OR_NAME] [--model.metric METRIC]
                             [--model.density.help CLASS_PATH_OR_NAME] [--model.density DENSITY] [--model.save_ply {true,false}] [--model.web_viewer {true,false}]
                             [--model.initialize_from INITIALIZE_FROM] [--model.renderer_output_types RENDERER_OUTPUT_TYPES] [--data CONFIG] --data.path PATH
                             [--data.parser.help CLASS_PATH_OR_NAME] [--data.parser PARSER] [--data.distributed {true,false}] [--data.undistort_image {true,false}] [--data.val_on_train {true,false}]
                             [--data.image_scale_factor IMAGE_SCALE_FACTOR] [--data.train_max_num_images_to_cache TRAIN_MAX_NUM_IMAGES_TO_CACHE]
                             [--data.val_max_num_images_to_cache VAL_MAX_NUM_IMAGES_TO_CACHE] [--data.test_max_num_images_to_cache TEST_MAX_NUM_IMAGES_TO_CACHE] [--data.num_workers NUM_WORKERS]
                             [--data.add_background_sphere {true,false}] [--data.background_sphere_center {points,cameras}] [--data.background_sphere_distance BACKGROUND_SPHERE_DISTANCE]
                             [--data.background_sphere_points BACKGROUND_SPHERE_POINTS] [--data.background_sphere_color {random,white}]
                             [--data.background_sphere_min_altitude BACKGROUND_SPHERE_MIN_ALTITUDE] [--data.camera_on_cpu {true,false}] [--data.image_on_cpu {true,false}]
                             [--data.image_uint8 {true,false}] [--data.async_caching {true,false}] [--max_steps MAX_STEPS] [--max_epochs MAX_EPOCHS] [--name NAME] [--version VERSION]
                             [--save_iterations SAVE_ITERATIONS] [--logger LOGGER] [--project PROJECT] [--output OUTPUT] [--float32_matmul_precision FLOAT32_MATMUL_PRECISION] [--viewer] [--save_val]
                             [--val_train] [--cache_all_images] [--ckpt_path CKPT_PATH]
main.py [options] fit: error: Parser key "config": File does not exist: '/data1/tantao/weizhi/code/gaussian-splatting-lightning/outputs/zhangzhou_largegs/-02_-02/config.yaml'

**Comment by yzslab** [comment]
Hi, have you remove the option `--logger wandb` when running training? If so, the `config.yaml` can be found in `outputs/<PROJECT_NAME>/<PARTITION_ID>/lightning_logs/`.
You need to update here to locate the correct `config.yaml`: https://github.com/yzslab/gaussian-splatting-lightning/blob/7eed95ac4acd5e27ae4930c16b417a466e443795/utils/finetune_pruned_partitions_v2.py#L23-L28

**Comment by tan-tao11** [comment]
Thanks, it works!
